config.yml

```diff
@@ -21,6 +21,7 @@ defaults:
     taskGroupActiveSetTableName:  TaskGroupActiveSets
     taskRequirementTableName:     TaskRequirement
     taskDependencyTableName:      TaskDependency
+    workerTypeTableName:          WorkerType
     provisionerTableName:         Provisioner
```

Comment:
I guess we forgot that here... we can change it... no harm in leaving the old table there... or we can just keep it, no worries :)